File: solutions/javascript/darts/1/darts.js
export const score = (x, y) => {
  const distance = Math.sqrt(x ** 2 + y ** 2);

  if (distance <= 1) {
    return 10; 
  }
  if (distance <= 5) {
    return 5; 
  }
  if (distance <= 10) {
    return 1;
  }
  
  return 0;    
};

File: solutions/kotlin/darts/1/src/main/kotlin/Darts.kt
import kotlin.math.sqrt

object Darts {
    fun score(x: Number, y: Number): Int {
        val xDouble = x.toDouble()
        val yDouble = y.toDouble()
        val distance = sqrt(xDouble * xDouble + yDouble * yDouble)

        return when {
            distance > 10 -> 0
            distance > 5  -> 1
            distance > 1  -> 5
            else          -> 10
        }
    }
}
